lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -139,6 +140,12 @@ export class EventHandler {
      */
     private handleAccountCacheChange(e: StorageEvent): void {
         try {
+            // Handle active account filter change
+            if (e.key?.includes(PersistentCacheKeys.ACTIVE_ACCOUNT_FILTERS)) {
```

Comment:
Confused a bit. We do not seem to compare for any change, but just the fact that the filters changed. Is that the intent?

Reply:
Yes, it's just a notice. Given that for localStorage the active account is shared across tabs, we're just notifying the other tabs/windows that the active account has changed so they can trigger UI updates on the other tabs. Any comparison required would in theory be made in `setActiveAccount`. I don't think there's a reason to add comparisons though.

Reply:
A bit related to what can trigger this event in my other comment, but does this mean there may not be meaningful account changes when this event triggers?

Reply:
It's possible that the account object in the cache won't change (for example, if there is a multi-tenant account and what changed was the active tenant, but both the previous active tenant profile and the new active tenant profile had already been cached).

Generally speaking if the event is triggered, the specific account/tenant profile that is active from MSAL's point of view changed. This means that the results of calling `getActiveAccount` before and after the event will be different.